warp/sim/integrator_xpbd.py

```diff
@@ -2811,9 +2811,6 @@ def simulate(self, model: Model, state_in: State, state_out: State, dt: float, c
                 if requires_grad:
                     particle_q = state_out.particle_q
                     particle_qd = state_out.particle_qd
-                else:
-                    particle_q = state_out.particle_q
-                    particle_qd = state_out.particle_qd
```

Comment:
Hi, thank you for the contribution!
It seems that `particle_q`, `particle_qd` will be None if `requires_grad` is False now - how about you remove the `if requires_grad` condition as well and always assign `particle_q`, `particle_qd`?

Reply:
Of course! Thank you for the feedback